### PR TITLE
feat(DateRangePicker): Add showWeekNumbers

### PR DESCRIPTION
**Pull request by TrinTragula**
Add the showWeekNumbers input to the DateRangePicker component.
I found the need for this feature, and realized that it was pretty much already implemented, the only thing missing was binding an input to the calendar used inside the component.

### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [x] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [x] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/date-range-picker/README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ With projected inputs:
 | overlaySettings  | OverlaySettings    | Changes the default overlay settings used by the `IgxDateRangePickerComponent`. | 
 | placeholder      | string             | Sets the `placeholder` for single-input `IgxDateRangePickerComponent`. |
 | weekStart        | number             | Sets the start day of the week. Can be assigned to a numeric value or to `WEEKDAYS` enum value. |
+| showWeekNumbers | number | Shows or hides week numbers. |
```
Comment (jackofdiamond5) [inline projects/igniteui-angular/src/lib/date-range-picker/README.md:123]:
```suggestion
| showWeekNumbers  | number             | Shows or hides week numbers. |
```

File: projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts [hunk 2]
```diff
@@ -257,6 +257,20 @@ export class IgxDateRangePickerComponent extends PickerBaseDirective
...
 
+    /**
+     * Show/hide week numbers
+     * 
+     * @remarks
+     * Default is `false`.
+     * 
+     * @example
+     * ```html
+     * <igx-date-range-picker [showWeekNumbers]="true"></igx-date-picker>
```
Comment (jackofdiamond5) [inline projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts:268]:
```suggestion
     * <igx-date-range-picker [showWeekNumbers]="true"></igx-date-range-picker>
```

**Comment by jackofdiamond5** [comment]
We might want to add a test in the `Rendering` section of the range picker's spec to check if the week numbers have been shown as expected.

**Review by kdinev — APPROVED** [review]

**Comment by kdinev** [comment]
@jackofdiamond5 I will merge the PR, but please pull a change updating the `CHANGELOG.md`.

**Pull request merged**